src/tango/xdp/fd_xdp_ctl_help

```diff
@@ -0,0 +1,117 @@
+
+Usage: %s [cmd] [cmd args] [cmd] [cmd args] ...
+
+Commands are:
+
+help
+- Prints this message
+
+init [WKSP]
+- Prepare an XDP environment by pinning the XDP redirect
```

Comment:
I wonder if we should point people here to `fd_xdp_redirect_prog`, so it's clear which redirect program you're referring to.